service/kvmconfig/v2/version_bundles.go

```diff
@@ -0,0 +1,119 @@
+package v2
+
+import (
+	"time"
+
+	"github.com/giantswarm/versionbundle"
+)
+
+func VersionBundles() []versionbundle.Bundle {
```

Comment:
v2 is special because it was the first version in which we introduced version bundles and some things went wrong, which is why we have multiple versions being served by one resource version. This will be removed eventually anyway. So just sorting this for consistency and transparency towards the new structure.